src/base/rss/rssarticle.h

```diff
@@ -49,18 +49,18 @@ namespace Rss
         Q_OBJECT
 
     public:
-        Article(Feed *parent, const QString &guid);
+        Article(Feed *parent);
 
         // Accessors
         bool hasAttachment() const;
-        const QString &guid() const;
+        const QString guid() const;
```

Comment:
ok. I am not a c++ programmer, so I guess there are some more similar mistakes I've made in there.
Should I fix it and re-submit a new pull request?